modules/pricer/src/test/java/com/opengamma/strata/pricer/capfloor/BlackIborCapletFloorletPeriodPricerTest.java

```diff
@@ -216,6 +219,24 @@ public void test_presentValue_afterPay() {
     assertEquals(computedFloorlet.getAmount(), 0d, NOTIONAL * TOL);
   }
 
+  public void test_presentValue_formula_shift() {
+    CurrencyAmount computedCaplet = PRICER.presentValue(CAPLET_LONG, RATES, SHIFTED_VOLS);
+    CurrencyAmount computedFloorlet = PRICER.presentValue(FLOORLET_SHORT, RATES, SHIFTED_VOLS);
+    double forward = RATES.iborIndexRates(EUR_EURIBOR_3M).rate(RATE_COMP.getObservation());
+    double expiry = SHIFTED_VOLS.relativeTime(CAPLET_LONG.getFixingDateTime());
+    double volatility = SHIFTED_VOLS.volatility(expiry, STRIKE, forward);
+    double df = RATES.discountFactor(EUR, CAPLET_LONG.getPaymentDate());
+    double shift = IborCapletFloorletDataSet.SHIFT;
+    double expectedCaplet = NOTIONAL * df * CAPLET_LONG.getYearFraction() *
+        BlackFormulaRepository.price(forward + shift, STRIKE + shift, expiry, volatility, true);
+    double expectedFloorlet = -NOTIONAL * df * CAPLET_LONG.getYearFraction() *
```

Comment:
Should be FLOORLET_SHORT.getYearFraction(). Even if the year fractions are the same it is more readable to use the Floorlet version.